index.html

```diff
@@ -67,9 +67,10 @@ <h2>Supporters</h2>
               <a title="Stickermule" href="httsp://stickermule.com/supports/opensourcedesign19-sponsorship" target="_blank" rel="noopener"><img src="images/supporters/stickermule.svg" height="40px" width="auto" alt="Stickermule Logo"/></a>
               <img src="images/supporters/wikimedia-germany.svg" height="50px" width="auto" alt="Wikimedia Germany Logo"/>
               <br>
-              <object type="image/svg+xml" data="https://opencollective.com/opensourcedesign/tiers/sponsor.svg?avatarHeight=70&width=150"></object>
+               <object type="image/svg+xml" data="https://opencollective.com/opensourcedesign/tiers/sponsor.svg?avatarHeight=70&width=150"></object>
```

Comment:
```suggestion
              <object type="image/svg+xml" data="https://opencollective.com/opensourcedesign/tiers/sponsor.svg?avatarHeight=70&width=150"></object>
```
Close, just one character off!